locale/de/LC_MESSAGES/django.po

```diff
@@ -43,30 +43,45 @@ msgid ""
 "    <p>Here's a rough outline of what we're trying to build:\n"
 "    "
 msgstr ""
+"\n"
+"    <p>Wir sind ein paar Typen aus der Schweiz mit der Idee ein System zu "
```

Comment:
"Typen" is extremely informal in german, not comparable to "Guys" in English. I suggest "Leute" instead